etc/init.d/zfs-zed.in

```diff
@@ -45,8 +39,7 @@ ZED_PIDFILE="@runstatedir@/$ZED_NAME.pid"
 
 do_depend()
 {
-	before zfs-import
-	after sysfs
+	after sysfs zfs-mount
```

Comment:
That is true, as long as `zfs-mount` runs in the boot runlevel and `zfs-zed` runs in the default runlevel.  I don't know of a way to enforce that, though, so it seems better to me to make the intention that `zfs-zed` depends on `zfs-mount` more explicitly clear.  I'm happy to defer to your expertise in this case.